.github/actions/java-test/action.yaml

```diff
@@ -38,9 +38,17 @@ runs:
       shell: bash
       run: |
         echo "JAVA_VERSION=${JAVA_VERSION}"
-        ./mvnw -B compile test-compile scalafix:scalafix -Psemanticdb -Djava.version=${JAVA_VERSION}
+        if [ $JAVA_VERSION == "1.8" ]; then
+          ./mvnw -B compile test-compile scalafix:scalafix -Psemanticdb -Djava.version=${JAVA_VERSION} -Dspotless.version=2.29.0
```

Comment:
is it OK to use 2.29.0 for Java 8?

Reply:
This line uses 2.29.0 for Java 8, isn't?

Reply:
I mean whether we'll run into the issue mentioned in the PR description if we use 2.29 for Java 8

Reply:
oh cool.